Directory.Packages.props

```diff
@@ -158,6 +158,7 @@
     <PackageVersion Include="Microsoft.Orleans.Sdk" Version="8.1.0-nightly.20240307.1" />
     <!-- Pinned version for Component Governance - Remove when root dependencies are updated -->
     <PackageVersion Include="Newtonsoft.Json" Version="13.0.3" />
+    <PackageVersion Include="Newtonsoft.Json.Schema" Version="3.0.15" />
```

Comment:
@mitchdenny - why did we choose to `Newtonsoft.Json.Schema` when we are already using `JsonSchema.Net` in this repo?